cnxarchive/__init__.py

```diff
@@ -53,6 +53,7 @@ def add_route(name, path, *args, **kwargs):
     add_route('export', '/exports/{ident_hash}.{type}{ignore:(/.*)?}')  # cnxarchive.views:get_export
     add_route('content-extras', '/extras/{ident_hash}')  # cnxarchive.views:get_extra
     add_route('search', '/search')  # cnxarchive.views:search
+    add_route('in-book-search', '/search/book/{ident_hash}')  # cnxarchive.views:in-book-search
```

Comment:
This should 'just work' with /search/{ident_hash}' , since the other search route never has a path component at all, only params